src/plugins/region_map/public/choropleth_layer.js

```diff
@@ -193,16 +193,14 @@ Make sure the file exists at that location.",
               values: { name: name },
             }
           );
-        } else if (e.config.url.includes('aws.a2z.com')) {
+        } else {
```

Comment:
This looks okay but any explicit reason to remove the ability to capture when the file cannot be downloaded vs this region map error. For tracking purposes.